internal/catalogmetadata/cache/cache_test.go

```diff
@@ -50,203 +56,196 @@ const (
 	}`
 )
 
-func TestCache(t *testing.T) {
-	t.Run("FetchCatalogContents", func(t *testing.T) {
```

Comment:
Similar to my other comment, I removed this unnecessary `t.Run` wrapper. When reviewing, it's much easier to see actual changes if you ignore whitespace.